apps/api/src/app/chat/chat.config.ts

```diff
@@ -13,10 +13,13 @@ export const assistantParams: AssistantCreateParams = {
 export const assistantConfig: AssistantConfigParams = {
   id: process.env['ASSISTANT_ID'] || '',
   params: assistantParams,
+  assistantPrefix: 'assistant-prefix',
```

Comment:
<img width="717" alt="image" src="https://github.com/user-attachments/assets/f0370e9b-7e2f-40c0-8a91-24eb005c55b0" />
Please check my screenshot, It looks like we have multiple instances instead of one with the prefix 🤔 

In my opinion we should have only once instance and decide if we would like have prefix or not.